content/applications/inventory_and_mrp/manufacturing/management/alternative_wc.rst

```diff
@@ -1,80 +1,65 @@
 ================================
-Using an alternative Work Center
+Alternative Work Centers
 ================================
 
-For many manufacturing companies, a common issue is to have to
-manufacture, at the same time, several products usually produced at the
-same work center. If in practice, employees can manufacture the goods at
-another work center, it is not that simple.
+Manufacturing several different products simultaneously at the same work center
+is a common issue for manufacturing companies. Although in practice, employees can 
+manufacture the goods at another work center instead of waiting for one to be available, 
+it's not always simple to have visibility on which work center is busy and plan resources 
+accordingly in real-time.
 
-You need to keep track of the job: which work center has been used and
-when, so you can schedule maintenance efficiently. With Odoo, you can
-configure it so you can keep tracking manufacturing orders and your
-employees have an alternative work center to use.
+With Odoo, you can configure alternative work centers to solve this issue. This automatically checks 
+which work center is busy and which equivalent ones are free, and schedules orders and maintenance accordingly.
 
-This way, your process becomes more efficient, and less non-productive
-time will be noticed.
+This way, your process becomes more efficient, as work center utilisation is optimised. 
 
 
 Configure your Work Centers
 ===========================
 
-The first thing to do is to go to the *Manufacturing* app settings.
-Then, enable the *Work Orders* feature and hit save.
+First, go to the Manufacturing app > Configuration> Settings. Then, enable the Work Orders
+feature and hit save.
 
 
-.. image:: media/alternative_wc_01.png
+.. image:: media/BOM_3.png
```

Comment:
Make sure that all new (or reworked) images have:
- a consistent file name
- the right location: place the file inside a folder of the same name as the doc file referring to it (pick the folder that makes the most sense if two pages are referring to the same image)
- an ALT tag

See https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images

Reply:
Also, please run `pngquant` on all these images to reduce their weight by a good 2/3rd.